src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/CodeStyleOption2`1.cs

```diff
@@ -129,9 +124,9 @@ private bool IsZeroOrOneValueOfEnum()
         public static CodeStyleOption2<T> FromXElement(XElement element)
         {
             var typeAttribute = element.Attribute("Type");
-            var valueAttribute = element.Attribute(nameof(Value));
-            var severityAttribute = element.Attribute(nameof(DiagnosticSeverity));
-            var version = (int?)element.Attribute(nameof(SerializationVersion));
+            var valueAttribute = element.Attribute("Value");
+            var severityAttribute = element.Attribute("DiagnosticSeverity");
+            var version = (int?)element.Attribute("SerializationVersion");
```

Comment:
i would prefer constants here.  so the reading/writing are sure to not have a mispelling issue.

Reply:
Makes sense.